standard/localisation.lua

```diff
@@ -0,0 +1,68 @@
+local Class = require('Module:Class')
+local Localisation = Class.new()
```

Comment:
```suggestion
local Localisation = Class.new()
```

You're not actually using this anywhere, so you could scrap this line and then use `Class.export(Localisation` at the bottom